CODEOWNERS

```diff
@@ -36,7 +36,7 @@
 /soc/riscv32/openisa*/                    @MaureenHelm
 /arch/x86/                                @andrewboie @wentongwu
 /arch/x86/core/                           @andrewboie
-/arch/x86/core/crt0.S                     @wentongwu @nashif
+/arch/x86/core/ia32/crt0.S                @wentongwu @nashif
```

Comment:
so if you are using ia32 for 32bit, what will you use for 64bit? hopefully not IA64 :)

Reply:
What else am I going to run on my Itanium smartwatch?